packages/replay-next/playwright/tests/source-search/should-expand-and-collapse-sources.ts

```diff
@@ -14,7 +14,7 @@ import { beforeEach } from "./beforeEach";
 beforeEach();
 
 test("should expand and collapse sources", async ({ page }, testInfo) => {
-  await toggleIncludeNodeModulesCheckbox(page, true);
+  await toggleExcludeNodeModulesCheckbox(page, true);
```

Comment:
This looks like a mistake. (We were saying toggle-include: true and now we're saying toggle-exclude: true)

Reply:
Oops, yeah I missed that. Should be fixed now